cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Code Quality issue: It's recommended to use ModelSerializer when you have a model associated with your serializer. It will simplify your code and automatically create fields for you. You can then remove the create and update methods as they will be handled by the ModelSerializer.